src/modes/behavior_manager.h

```diff
@@ -0,0 +1,218 @@
+#ifndef BEHAVIOR_MANAGER_H
+#define BEHAVIOR_MANAGER_H
+
+//
+// note: this code is included as-is by:
+//  - user/indexable_functions.h
+//  - simulator/src/mode-simulator.cpp
+//
+
+void power_on_sequence() {
+  auto manager = get_context();
+
+  // power on
+  pinMode(LED_POWER_PIN, OUTPUT);
+  digitalWrite(LED_POWER_PIN, HIGH);
+
+  // initialize the strip object
+  manager.strip.begin();
+  manager.strip.clear();
+  manager.strip.show();  // Turn OFF all pixels ASAP
+  manager.strip.setBrightness(BRIGHTNESS);
```

Comment:
the brightness set should only be a made with the function, and at this point the value is maybe not set.
So this line should be removed :)

Reply:
This comes from the previously existing `power_on_sequence()` previously written by someone else in `user/indexable_functions.cpp` and after testing with & without this line, it seems like it is necessary to be present (or else, the strip always starts with the maximal brightness, then on the first adjustment with the button, changes to `BRIGHTNESS` then animates)

I can modify this, but as this is the previously existing behavior and that haven't changed here, I'm not sure it should be the purpose of this PR. Marking this as solved, feel free to unsolve this if it needs further discussion.